dsgrid/config/dataset_config.py

```diff
@@ -1,3 +1,4 @@
+import abc
```

Comment:
Why is this new import not showing up in another diff line where it is used?

Reply:
This needs to be deleted.